airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlQueryUtils.java

```diff
@@ -141,4 +216,22 @@ private static List<JsonNode> getTableEstimate(final JdbcDatabase database, fina
     return jsonNodes;
   }
 
+  public static boolean isAnyStreamIncrementalSyncMode(final ConfiguredAirbyteCatalog catalog) {
```

Comment:
This method is exactly same as `shouldUseCDC` method in `AirbyteDebeziumHandler`. lets just rename the one in `AirbyteDebeziumHandler` and delete this